server/seeder.js

```diff
@@ -0,0 +1,45 @@
+const faker = require('faker');
+const moment = require('moment');
+const mongoose = require('mongoose');
+const db = require('./database/db');
+
+// Load Profile Model
+const ReservedDates = require('./models/ReserveDates');
+
+const seeder = () => {
+
+  // Droping existing sample data
+  ReservedDates.remove({}, ()=>{
+    console.log('Removed Sample Data')
+  })
+// Loop for each listing
+  for(let j = 1; j <= 100; j++){
+
+    let bookedDays = []
+
+    // Loop 
+    for(let i = 0; i < 50; i++){
+      let randomDate = faker.date.between('2018-12-01', '2019-06-01');
+      let cleanDate = moment(randomDate).format('L')
+      bookedDays.push(cleanDate)
+    };
+    
+    const newReservedDates = new ReservedDates({
+      listing_id: j,
+      bookedDates: bookedDays,
+    });
+    
+    newReservedDates.save().then((res) => {
+  
+      console.log('Sample data has been saved to DB')
+    });
+  };
+};
+
+module.exports = seeder;
```

Comment:
Maybe run seeder in here? Is there a reason you need to export it? eventually, you'll be writing a npm script like `npm run seed-database` which will do something like `node ./server/seeder.js`

Also since the seeder file is specific to the database, consider moving it to the database folder.

Reply:
Yup, took care of that